src/components/Image/DetailV2/ImageMeta.tsx

```diff
@@ -136,6 +136,13 @@ export function ImageMeta({ imageId }: { imageId: number }) {
           </div>
         </>
       )}
+      {/* TODO: trying out few render ux   */}
+      {remixedFromImageId && (
+        <div className="flex flex-col">
+          <Text className="font-semibold">Remixed From Image ID</Text>
+          <Text className="leading-snug">{remixedFromImageId}</Text>
```

Comment:
Maybe this is still in progress, but I think we can change wording here and link to the actual image instead
```suggestion
          <Link href={`/images/${remixedFromImageId}`} passHref>
            <Anchor className="font-semibold">Original Image</Text>
          </Link>
```
Maybe this works, we could load a small thumbnail from the original image as well probably, not sure how good it'll look in the meta popover tho 😬

Reply:
Might be good to perform a query et all - Technically, the image could've been removed.